helm/chaos-mesh/templates/controller-manager-rbac.yaml

```diff
@@ -144,7 +141,9 @@ kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: {{ .Release.Name }}-chaos-controller-manager-target-namespace
+{{- if not .Values.clusterScoped }}
   namespace: {{ .Values.controllerManager.targetNamespace | quote }}
+{{- end }}
```

Comment:
I think I could not accept the changes except these. As we have talked on slack, here is indeed the dirty thing that I want to clean up. 😆

In this controller-manager-rbac.yaml file,  here are three `Role` or `ClusterRole`:

- `{{ .Release.Name }}-chaos-controller-manager-control-plane` represents the permission for the namespace which installed the chaos-mesh; It requires permissions on `serivces` and `configmaps`; it always use `RoleBinding` with service acount
- `{{ .Release.Name }}-chaos-controller-manager-cluster-level` represents the permission for the cluser-level, no matter with the `.Value.clusterScoped`, it always require the permission on `nodes` and `subjectaccessreviews`; it always use `ClueterRoleBinding` with service acount
- `{{ .Release.Name }}-chaos-controller-manager-target-namespace` represents the permission for the namespace which will be injected chaos; if `.Value.clusterScoped = true` it bind service account by `ClusterRoleBinding`, other with with `RoleBinding`

So we could not remove the `{{ .Release.Name }}-chaos-controller-manager-control-plane` even with `.Value.clusterScope = true`

Reply:
If `.Value.clusterScope = true` then what ever permissions we are giving in `{{ .Release.Name }}-chaos-controller-manager-control-plane`  `Role` to resources like `configmaps`, `services` and `subjectaccessreviews`, these we are already giving these permission by `ClusterRole` as well.

Reply:
We are giving the following permission in this `Role`:

```
rules:
  - apiGroups: [ "" ]
    resources: [ "configmaps", "services" ]
    verbs: [ "get", "list", "watch" ]
  - apiGroups: [ "authorization.k8s.io" ]
    resources:
      - subjectaccessreviews
    verbs: [ "create" ]
```

and these permissions are also part of two cluster roles as well.

Reply:
Am I missing some thing ? 🤔